app/controllers/blueprints_controller.rb

```diff
@@ -6,9 +6,9 @@ class BlueprintsController < ApplicationController
   def buy
     if params[:loader] && params[:type] && current_user.location.research_station?
       if params[:type] == 'item'
-        price = get_item_attribute(params[:loader], 'price') * 20 rescue nil
+        price = Item.get_attribute(params[:loader], 'price') * 20 rescue nil
```

Comment:
Could that also be changed to `:price` ?

Reply:
yep -- just didn't want to change all the things :) 

`Item.get_attribute` now takes symbols or strings

Reply:
Could you change it? Would be awesome! ❤️